logspout.yml

```diff
@@ -0,0 +1,18 @@
+version: '2'
+
+services:
+  logspout:
+    build: logspout/
+    volumes:
+      # security concern:
```

Comment:
You can probably remove that comment, it's more oriented towards global Docker usage than related to this project.